src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -115,17 +111,30 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
               </>
             )}
 
-            {this.maybeShowGenes && (
-              <>
-                <Media at="xs">
-                  {bioLen < MAX_CHARS.xs && <Genes artist={artist} />}
-                </Media>
-                <Media greaterThan="xs">
-                  {bioLen < MAX_CHARS.default && <Genes artist={artist} />}
-                </Media>
-                <Spacer mb={1} />
-              </>
-            )}
+            <>
+              <Media at="xs">
+                {bioLen < MAX_CHARS.xs ? (
+                  <>
+                    <Genes artist={artist} />
```

Comment:
Is this right? It looks like before, this would have been guarded by a `showGenes`, but now it's not (if the `bioLen < MAX`).

Reply:
This is covered further down. The logic is 
- if bio len is less than a certain size, show, independent of other checks 
- if not, then check to see if read more has been clicked and other conditions are met and hide until those conditions are true